static/src/javascripts/projects/common/modules/experiments/ab.js

```diff
@@ -137,7 +137,7 @@ export const getEpicTestToRun = memoize(
                             expectedVariant: result
                                 ? result.variantToRun.id
                                 : '',
-                            expectedCampaignId: result ? result.campaignId : '',
+                            expectedCampaignId: result ? ((result: any).campaignId)  : '', // type cast as must be EpicABTest at this poimt
```

Comment:
could you use `AcquisitionsABTest` to avoid having to cast here?

Reply:
Yes, although we'd have to add some dummy values to the `RemoteEpicVariants` test definition (campaignId really). But that is okay and better than casting any probably.